### DISCUSSION TITLE
elastalert.util.EAException: Error loading file /opt/elastalert/rules/active01.yaml: Error initiating alert ['workwechat']: Could not import module workwechat: not enough values to unpack (expected 2, got 1)

### DISCUSSION THREAD

**Discussion by asuoghs** [jertel/elastalert2#1379] [Q&A] [answered]
STOP - You are probably in the wrong place! The majority of topics are better suited for the Discussion forum. You can access this area by clicking The Discussions link above. Please search the discussions area first, for keywords that could be associated with the problem you are experiencing. If you do not see an existing discussion, please open a new discussion and include sufficient details for someone in the community to help you.

If you are confident you have discovered a legitimate issue, attach logs and reproduction steps to this issue. Failure to provide sufficient information will likely cause this issue to go stale and eventually be deleted.

**Comment by jertel** [comment, marked as answer]
Please read #11 before requesting support. 

Most likely you are attempting to use a new module that's not yet been released.
